front-end/packages/robustar/src/views/ImageList.vue

```diff
@@ -315,6 +343,11 @@ export default {
         { text: 'Incorrectly Classified', value: this.$route.params.split + '_incorrect' },
       ];
     },
+    accuarcy() {
+      const allImageLength = this.testImageList[0]
+      const correctImageLength = this.testImageList[1]
```

Comment:
We can use array destructuring here for simplicity: `const [allImageLength, correctImageLength] = this.testImageList`

Reply:
However, const [allImageLength, correctImageLength] = this.testImageList will result in "TypeError: object is not iterable" issue. I tried to change it to const {allImageLength, correctImageLength} and some other ways and it turns out to be this is the only way to represent the accuracy, though it was ugly.